episodes/01-introduction.md

```diff
@@ -1,101 +1,94 @@
 ---
-title: "Model Coupling"
-teaching: 10
-exercises: 2
+title: "Introduction to model coupling"
+teaching: 30
+exercises: 20
 ---
 
 :::::::::::::::::::::::::::::::::::::: questions 
 
-- How do you write a lesson using Markdown and `{sandpaper}`?
+- What is meant by a "model"?
+- What is the difference between a mathematical model and a submodel?
+- What is model coupling?
+- Why would we want to couple simulation models?
 
 ::::::::::::::::::::::::::::::::::::::::::::::::
 
 ::::::::::::::::::::::::::::::::::::: objectives
 
-- Explain how to use markdown with The Carpentries Workbench
-- Demonstrate how to include pieces of code, figures, and nested challenge blocks
+- Explain the benefits of coupling simulation models
 
 ::::::::::::::::::::::::::::::::::::::::::::::::
 
 ## Introduction
+Computer simulations are widely used in modern science to understand and predict the behaviour of natural systems. The computer program that simulates the system is referred to as a "model" of that system. Some models may be very simple e.g. describing the evolution of a 1 dimensional system. Others may be highly complex, involving many different physical processes. A well known use of computer simulation is in the field of weather forecasting, in which a very complex model composed of a multitude of different physical processes must be simulated by a computer program, generally requiring powerful supercomputers. The computer program that simulates a given physical system is known as a _model_ of that system. A given climate simulation code is a _model_ of the Earth's climate, for example.
```

Comment:
"The computer program that simulates the system is referred to as a "model" of that system."
"The computer program that simulates a given physical system is known as a _model_ of that system."
These two sentences are almost identical, maybe skip one?

Reply:
Yes one of these should go. I'll merge both into the first occurence